### PR TITLE
Add passenger monitor

### PR DESCRIPTION
**Pull request by newzac**
Description and Impact
----------------------
> What impact does this change have for production?

Deploy Plan
-----------
> Does Platform Operations need to know anything special about this deploy? Are migrations present?

Rollback Plan
-------------
* To roll back this change, revert the merge with: `git revert -m 1 MERGE_SHA` and perform another deploy.

URLs
----
> [IS-3994](https://sportngin.atlassian.net/browse/IS-3994)

QA Plan
-------

### CODE REVIEW

File: attributes/default.rb [hunk 1]
```diff
@@ -50,6 +50,10 @@
 default[:ruby_wrapper][:heap_free_min] = 100000
 default[:ruby_wrapper][:extra_env_vars] = {}
 
+# Passenger Monitor
+default[:passenger][:monitor][:passenger_soft_memory_limit] = 250
```
Comment (bbergstrom) [inline attributes/default.rb]: `passenger_` in the sub keys is a bit redundant.

File: recipes/passenger-monitor.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  action :create
+  variables({
+    :soft_memory_limit => node[:passenger][:monitor][:passenger_soft_memory_limit], 
+    :hard_memory_limit => node[:passenger][:monitor][:passenger_hard_memory_limit],  
+    :processed_limit => node[:passenger][:monitor][:passenger_requests_processed_limit]
+  })
+end
+
+# * * * * * /usr/local/bin/tst_passenger_monitor;
+cron "tst_passenger_monitor_ngin" do
```
Comment (bbergstrom) [inline recipes/passenger-monitor.rb]: Generalize the bits like `tst` and `ngin`

**Comment by newzac** [comment]
:octocat: Merged into staging.2016.02.22.

**Comment by bbergstrom** [comment]
CR :+1:

**Comment by anfleene** [comment]
:+1:

**Pull request merged**